ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -867,7 +844,7 @@ static bool TryGetVirtualizationRoot(
 {
     PerfSample findRootSample(perfTracer, PrjFSPerfCounter_VnodeOp_GetVirtualizationRoot);
         
-    *vnodeFsidInode = Vnode_GetFsidAndInode(vnode, context);
+    *vnodeFsidInode = Vnode_GetFsidAndInode(vnode, context, true);
```

Comment:
Could you add a small comment here (and in `ShouldHandleFileOpEvent` where we call `Vnode_GetFsidAndInode`) that describes why `true` is being passed in?

Reply:
Done!